.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
✨ To keep it pretty, line by line,
Our code will shine, and be so fine.

We keep it pretty, clean and neat,
Code that's sloppy is hard to beat.

We keep it pretty & indentated,
Writing comments, annotated

We keep it pretty so we can read
and it's so late I need to sleep 🎶✌🏻